db/db.go

```diff
@@ -128,3 +128,20 @@ func (p *ClaDB) MigrateDB(migrateSourceURL string) (err error) {
 	}
 	return
 }
+
+const sqlInsertAuthorMissing = `INSERT INTO unsigned_pr
+		(repo_owner, repo_name, pr_number, login_name, ClaVersion, CheckedAt)
+		VALUES ($1, $2, $3, $4, $5, $6) ON CONFLICT DO NOTHING`
```

Comment:
Just thinking out loud, might be good to have a table with the PR, and a child table with the signatures (we can add or remove people from that list, because it can and very will be more than 1 person).

Reply:
Done: e83fcc8